tests/kernel/install_ltp.pm

```diff
@@ -215,12 +215,8 @@ sub add_ltp_repo {
         $arch = "_PowerPC"  if is_ppc64le();
         $arch = "_zSystems" if is_s390x();
 
-        if (want_stable) {
-            $repo = "https://download.opensuse.org/repositories/benchmark/openSUSE_Factory$arch/";
-        } else {
-            $arch = ((is_x86_64 || is_aarch64) ? "Tumbleweed" : "Factory") . $arch;
-            $repo = "https://download.opensuse.org/repositories/benchmark:/ltp:/devel/openSUSE_$arch/";
-        }
+        $arch = ((is_x86_64 || is_aarch64) ? "Tumbleweed" : "Factory") . $arch;
+        $repo = "https://download.opensuse.org/repositories/benchmark:/ltp:/devel/openSUSE_$arch/";
```

Comment:
Is function `want_stable` used anywhere else?  I don't see it, maybe we can remove it as well.

Reply:
Well, for SLES It's the default for `is_sle && is_released`.